usagov/vote-gov
# Replace Massachusetts registration website link

https://bixal-projects.atlassian.net/browse/VOTE-960

Testing Steps:
1. Check out VOTE-960/fix-broken-link-ma
2. Run this command: npm run cy:proofer
3. Verify the link shown in the ticket works when clicked in the browser.
